core/vm/stack_table.go

```diff
@@ -16,27 +16,14 @@
 
 package vm
 
-import (
-	"github.com/ethereum/go-ethereum/params"
-)
-
 func minSwapStack(n int) int {
 	return minStack(n, n)
 }
-func maxSwapStack(n int) int {
-	return maxStack(n, n)
-}
 
 func minDupStack(n int) int {
 	return minStack(n, n+1)
 }
-func maxDupStack(n int) int {
-	return maxStack(n, n+1)
-}
 
-func maxStack(pop, push int) int {
-	return int(params.StackLimit) + pop - push
-}
 func minStack(pops, push int) int {
```

Comment:
I think at this point it makes sense to get rid of `minStack` / `minDupStack` / `minSwapStack`  helpers, too.